test/modules/packages/distributedIters/perf/benchmarkDistributedIters.perfexecopts

```diff
@@ -0,0 +1,2 @@
+--test=uniform --mode=guided --n=10000 # benchmarkDistributeIters-guided.perfkeys
```

Comment:
I think we can do without the distinct 'perfkeys' files here and instead just do something like "# distributedGuided". See the following perfexecopts for examples:

[`linear algebra`](https://github.com/chapel-lang/chapel/blob/master/test/modules/packages/linearalgebra/performance/transpose-perf.perfexecopts)
[`sort`](https://github.com/chapel-lang/chapel/blob/master/test/modules/packages/Sort/performance/performance.perfexecopts)